#72 Added support for reusing containers (#73)
exasol/exasol-testcontainers: doc/user_guide/user_guide.md

sudo apt install rng-tools
```

## Reusing Container for Speedup

Exasol testcontainers can reuse the running container to avoid the startup delay.

> for that you need to append `withReuse(true)` at container creation.

Reviewer: ```suggestion
For that you need to append `withReuse(true)` at container creation.
```

Reviewer: Could you please add short example code below this line?

```java
             new ExasolContainer<>()
             // ...
            .withReuse(true);
```

> In addition you have to add `testcontainers.reuse.enable=true` to `~/.testcontainers.properties` on your machine.

Reviewer: ```suggestion

In addition you have to add `testcontainers.reuse.enable=true` to `~/.testcontainers.properties` on your machine. This only enables the reuse only for the local development but not for continuous integration (CI).
```

Author: ```suggestion
In addition you have to add `testcontainers.reuse.enable=true` to `~/.testcontainers.properties` on your machine. This only enables the reuse for the local development but not for continuous integration (CI).
```

> Then Exasol testcontainers will keep your container running after the tests and reuse if if possible.

Reviewer: ```suggestion
Then Exasol testcontainers will keep your container running after the tests and reuse across the tests.
```

> Since testcontainers will not terminate the container any more you have to care yourself. 

Reviewer: ```suggestion
Since testcontainers will not terminate the container any more you have to manually terminate it. 
```

> You can find the container id using `docker ps` and terminate it using `docker rm -f <CONTAINER-ID>`.  

Reviewer: ```suggestion
You can find the container id using `docker ps` command and terminate it by running `docker rm -f <CONTAINER-ID>`.  
```
